Vagrantfile

```diff
@@ -18,12 +18,12 @@ module KatelloDeploy
       :libvirt    => 'http://m0dlx.com/files/foreman/boxes/centos64.box'
     },
     :centos7 => {
-      :box_name   => 'centos7',
+      :box_name   => 'centos7_1',
```

Comment:
i wanted to change the name in order to force everyone to download it, especially anyone that may have gotten the 'bad' 7.0 box we had been pointing to.  We could provide steps to remove it i guess instead?